src/database/models/version_item.rs

```diff
@@ -621,12 +621,12 @@ impl Version {
                     ARRAY_AGG(DISTINCT pt.name) filter (where pt.name is not null) project_types,
                     ARRAY_AGG(DISTINCT g.slug) filter (where g.slug is not null) games
                 FROM versions v
-                INNER JOIN loaders_versions lv ON v.id = lv.version_id
-                INNER JOIN loaders l ON lv.loader_id = l.id
-                INNER JOIN loaders_project_types lpt ON lpt.joining_loader_id = l.id
-                INNER JOIN project_types pt ON pt.id = lpt.joining_project_type_id
-                INNER JOIN loaders_project_types_games lptg ON lptg.loader_id = l.id AND lptg.project_type_id = pt.id
-                INNER JOIN games g ON lptg.game_id = g.id
+                LEFT JOIN loaders_versions lv ON v.id = lv.version_id
+                LEFT JOIN loaders l ON lv.loader_id = l.id
+                LEFT JOIN loaders_project_types lpt ON lpt.joining_loader_id = l.id
+                LEFT JOIN project_types pt ON pt.id = lpt.joining_project_type_id
+                LEFT JOIN loaders_project_types_games lptg ON lptg.loader_id = l.id AND lptg.project_type_id = pt.id
+                LEFT JOIN games g ON lptg.game_id = g.id
```

Comment:
any reason these were converted to left joins? all of these are mandatory fields so not sure why- a version will always have a loader no matter what